PendingReleaseNotes.md

```diff
@@ -6,4 +6,5 @@
 ## Features
 
 
-- Previously, only the latest version of helm was tested and the docs stated only version 3.x of helm as a prerequisite. Now rook supports the six most recent minor versions of helm along with their their patch updates. Explicitly, helm versions 3.13 and newer are supported.
\ No newline at end of file
+- Previously, only the latest version of helm was tested and the docs stated only version 3.x of helm as a prerequisite. Now rook supports the six most recent minor versions of helm along with their their patch updates. Explicitly, helm versions 3.13 and newer are supported.
+- Add support for `GatewayAPI` `HTTPRoute` as an alternative to `Ingress` for Ceph dashboard and `CephObjectStore`.
```

Comment:
In what release would you like this feature? These pending release notes are for v1.18, which is about a month out. This feature is small enough I could imagine we could backport to 1.17.x if you desire. In that's the preference, please  remove this comment.